packages/flutter_tools/lib/src/fuchsia/fuchsia_ffx.dart

```diff
@@ -0,0 +1,101 @@
+// Copyright 2014 The Flutter Authors. All rights reserved.
```

Comment:
2021 since this is a new file

Reply:
flutter/flutter repo should have 2014 on all files.

Reply:
Oh interesting, was not aware of that.